frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -218,6 +218,7 @@ export const InteractiveEmbeddingOptionCard = ({
           label={isInteractiveEmbeddingEnabled ? t`Enabled` : t`Disabled`}
           ml="auto"
           labelPosition="left"
+          disabled={!isEE}
```

Comment:
use the same condition as the `Configure` button. That is using EE with `embedding` flag.

https://github.com/metabase/metabase/blob/8ff9d9a1aa63accdbadcda509aa6a0dad588ba49/enterprise/frontend/src/metabase-enterprise/embedding/index.js#L7-L8